javaprocessor/pom.xml

```diff
@@ -74,15 +74,14 @@
     </build>
     <dependencies>
         <dependency>
-            <groupId>sawtooth</groupId>
-            <artifactId>sdk</artifactId>
-            <version>1.0-SNAPSHOT</version>
+            <groupId>org.hyperledger.sawtooth</groupId>
+            <artifactId>sawtooth-sdk-transaction-processor</artifactId>
+            <version>v0.1.2</version>
```

Comment:
Does this mean you are using version 1.2 of the SDK?
Is that the same as Sawtooth nightly (1.2)?
Because normally you would use 1.1 of the SDK.

Reply:
The v1.1 SDK available in maven repo is compiled to JDK11. Whereas in xenial the highest official version is JDK8. And in bionic the highest official version is JDK10. OTOH, SDK 1.2 works fine with default-jre in xenial as well as bionic, without throwing class version mismatch errors.
Seems to me v1.1 SDK in maven repo is incompatible/broken.